.gitignore

```diff
@@ -9,6 +9,9 @@ __pycache__/
 # C extensions
 *.so
 
+# IDE files
```

Comment:
I could not get the .gitignore to work from inside `.idea`... Not sure why, but other examples on I have seen just put it here.